### DISCUSSION TITLE
x86 OS X Support

### DISCUSSION THREAD

**Discussion by k00000han** [modular/modular#1599] [General]
I've heard about **Mojo** just now and start to explore everything about this new amazing language. I managed to read that **Mojo** doesn't support x86 architecture in **OS X** based computers, isn't it? I've seen in requirements: "Apple silicon (M1 or M2 processor)". Will Modular add support for **x86** based Mac? 
I would love to start exploring this language in practice but i can't bcs I have Intel based Mac🤒
<img width="404" alt="Screenshot 2024-01-05 at 6 15 05 " src="https://github.com/modularml/mojo/assets/133035570/053ceaea-6636-43b3-9bf2-26765448645a">

**Comment by Sharktheone** [comment]
Yes, there will be macOS Intel support at some point. This is tracked as #1092, so check this one out for updates.
